io.github.andrewmkiss.Demeter.yml

```diff
@@ -0,0 +1,190 @@
+app-id: io.github.andrewmkiss.Demeter
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: dathena
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission.

Reply:
Yes, I kept these broad with host because the software will open files that are located in different locations. Sometimes I have these installed in $HOME, but often these are on network mounts as well. Of the options in the flatpak documentation, this seemed the most relevant because we don't always know where those network mounts are located